src/modules/ekf2/EKF2.cpp

```diff
@@ -1990,7 +1990,7 @@ void EKF2::PublishYawEstimatorStatus(const hrt_abstime &timestamp)
 #if defined(CONFIG_EKF2_WIND)
 void EKF2::PublishWindEstimate(const hrt_abstime &timestamp)
 {
-	if (_ekf.get_wind_status()) {
+	if (_ekf.get_wind_status() || _ekf._external_wind_init) {
```

Comment:
Really `_external_wind_init` should be private. 

After that does it make sense to push it into get_wind_status()?